espnet/nets/pytorch_backend/transformer/dynamic_conv.py

```diff
@@ -0,0 +1,99 @@
+import math
+import logging
+
+import numpy
+import torch
+import torch.nn.functional as F
+from torch import nn
+
+
+MIN_VALUE = float(numpy.finfo(numpy.float32).min)
+
+
+class DynamicConvolution(nn.Module):
+    """Dynamic Convolution layer
+    This implementation is based on https://github.com/pytorch/fairseq/tree/master/fairseq
+
+    :param int whare: the number of kernel of convolution
+    :param int n_feat: the number of features
+    :param float dropout_rate: dropout rate
```

Comment:
Can you add all the parameters even if it looks obvious? It hepls our sphinx doc.